src/mcdu-server/client/src/McduScreen.css

```diff
@@ -0,0 +1,171 @@
+@font-face {
+    font-family: "HoneywellMCDU";
+    src: url("HoneywellMCDU.ttf");
+}
+
+@font-face {
+    font-family: "HoneywellMCDUSmall";
+    src: url("HoneywellMCDUSmall.ttf");
+}
+
+.screen {
+    --displayBackground: rgba(13, 20, 35, 1);
+    --displaySeparator: #3c3c3c;
+    --displaySeparatorLight: #aaaaaa;
+
+    --displayWhite: #ffffff;
+    --displayGrey: #787878;
+    --displayDarkGrey: #b3b3b3;
+    --displayLightGrey: lightgray;
+    --displayAmber: #e68000;
+    --displayCyan: #00ffff;
+    --displayGreen: #00ff00;
+    --displayMagenta: #ff94ff;
+    --displayRed: #ff0000;
+    --displayYellow: #ffff00;
```

Comment:
Is it too cheeky to include these defs from `src\instruments\src\Common`?

Reply:
I'm not sure that's the best idea. If someone makes changes to that file it's unlikely they will test this too.